src/hmac/hmac_sha1.c

```diff
@@ -38,25 +40,23 @@ void hmac_sha1(const uint8_t *k,  /* secret key */
 	       uint8_t *out,      /* output buffer, at least "t" bytes */
 	       size_t   t)
 {
-#ifdef USE_OPENSSL
 	(void)t;
 
+#if defined (USE_OPENSSL)
 	if (!HMAC(EVP_sha1(), k, (int)lk, d, ld, out, NULL))
 		ERR_clear_error();
 #elif defined (__APPLE__)
-	(void)t;
-
 	CCHmac(kCCHmacAlgSHA1, k, lk, d, ld, out);
+#elif defined (USE_MBEDTLS)
+	mbedtls_md_hmac(mbedtls_md_info_from_type(MBEDTLS_MD_SHA1),
+			k, lk, d, ld, out);
```

Comment:
consider the order here ... with this setup Apple CommonCrypto will always be selected
before MbedTLS.